test-data/unit/check-generics.test

```diff
@@ -766,7 +766,7 @@ if not isinstance(s, str):
 
 z = None # type: TNode # Same as TNode[Any]
 z.x
-z.foo() # E: Some element of union has no attribute "foo"
+z.foo() # Any simplyfies Union to Any
```

Comment:
Actually, Union[Any, int] is *not* the same as Any. This is subtle (I got it wrong in an early version of typing.py) but it's important, since when it's an int, it can't be used as a string. See PR #2197